### DISCUSSION TITLE
where demo source?

### DISCUSSION THREAD

**Discussion by kuridaik** [cloudscape-design/components#61] [Q&A] [answered]
I cannot find demo's code. where?

**Comment by michaeldowseza** [comment]
Hi @kuridaik ,

Currently the source code for the demos on the website have not been open sourced but we have it on our radar.

**Reply by everdrone** [reply]
Are you planning on open sourcing the demos?
That would be super useful to quickly get a grasp of the basics and the applied patterns described in the documentation

**Reply by 6be709c0** [reply]
Looks like some demos are available here https://github.com/cloudscape-design/components/blob/main/pages/app-layout/with-table.page.tsx
From https://github.com/cloudscape-design/components/discussions/111

**Reply by BadBeatBuck** [reply]
Demos with no source code are not very helpful.
Do the demos come from the /pages folder?

**Reply by llermaly** [reply]
+1 , would be awesome to have the demos code. There is no snippet example for the App Layout component, had to visit the source code.

**Reply by matthewepler** [reply]
+1 much needed. I was able to copy/paste from the source files in the browser, but a lot has to be edited to get them to work (import package names, style document locations, etc). Still, there is no package.json or SCSS files so it only gets you so far. 

User stories:
"As a developer, I want to get a proof-of-concept up and running as quickly as possible so that I can iterate faster with my user research."
"As a developer, I want to scaffold the pages/views in my app with template code so that I can get my project going quickly and without guesswork."

**Reply by zhangxiang93** [reply]
+1 much needed

**Reply by tbelmega** [reply]
+1 I'd love to migrate our new aws solution from awsui to cloudscape before reinvent, but I don't have time to study all available components and patterns and puzzle them together myself. The demo pages look really great, with the source code we could understand quickly how those components are supposed to be used together.
EDIT: To be fair, your documentation with playground and usage code examples is pretty great and helpful in that regard! Great job!

**Reply by ibrahimcesar** [reply]
+1 Would be specially useful to see some components not yet covered such as light/dark mode and proper footers for pages.

**Comment by gethinwebster** [comment, marked as answer]
We've now published the demo source code: it's available at https://github.com/cloudscape-design/demos
